src/System.Linq/tests/ConcatTests.cs

```diff
@@ -4,68 +4,48 @@
 
 using System;
 using System.Collections.Generic;
+using System.Diagnostics;
 using Xunit;
 
 namespace System.Linq.Tests
 {
     public class ConcatTests : EnumerableTests
     {
-        [Fact]
-        public void SameResultsRepeatCallsIntQuery()
-        {
-            var q1 = from x1 in new int?[] { 2, 3, null, 2, null, 4, 5 }
-                     select x1;
-            var q2 = from x2 in new int?[] { 1, 9, null, 4 }
```

Comment:
`InlineData` does not work with `Nullable<int>` since the compiler says it's not a constant expression, so I had to change it to a plain int.